README.md

```diff
@@ -1,89 +1,494 @@
+and remember to track your time so you don't over-work.
+
+Six hours either sounds like _little_
+(_if you are used to working 12h per day at a minimum wage job_),
+or a _lot_ if you have attempted to _focus_ without distractions for 6 hours.
+You are _not_ expected to work _continuously_ for 6 hours.
+You _are_ expected to keep _track_ of your time
+
+> <sup>1</sup>If you haven't yet read the book **Deep Work** by Cal Newport
+make it a priority to read it and understand all the concepts
+***before*** starting the ACT.
+
+
+### Q: How to _Communicate_?
+
+
+1. **GitHub** - GitHub is our
+[single source of truth](https://en.wikipedia.org/wiki/Single_source_of_truth).
+We have a _strong_ preference for communicating via GitHub.
+This ensures that _anyone_ can see what everyone _else_ is learning
+or working on and can _help_ when we get stuck.
+If in doubt, open an issue in the repository you are working on.
+
+
+2. Daily Standup Meetings - We have a
+[daily standup meeting](https://en.wikipedia.org/wiki/Stand-up_meeting)
+at **09:00** each day of the work week (_Monday - Friday_) <br />
+_Everyone_ attends via **Zoom** video conference:
+https://zoom.us/j/3371337133 <br />
+Join the call at **08:59**.
+
+> We keep our individual updates _brief_.
+The point is just to describe in a few words what
+you worked on the _previous_ day
+and what your plan is for _today_.
+We mention an issue number and project name e.g: <br />
+"_Yesterday I worked on Autocomplete issue
+[#224](https://github.com/dwyl/app/issues/224).
+I realised that there is more work to do
+before Autocomplete can work,
+so today I am working on Basic tagging
+[#245](https://github.com/dwyl/app/issues/224)_"
+
+> If you are _stuck_ with the task
+you are _currently_ working on,
+make that clear by writing a **comment**
+in the issue you are working on
+and draw people's attention to it in the standup.
+
+If for any reason you are _unable_ to attend the daily standup,
+send a message with your update on the **@dwyl** Signal channel.
+
+
+3. Signal - We use signal messenger https://signal.org
+to _coordinate_ and ask "_logistical_" questions.
+Signal is ephemeral so we _never_ use it
+to ask technical questions,
+those are _always_ asked/answered on GitHub
+so that they're indexed by Google
+and thus helpful to _everyone_.
+
+
 
 ### Q: How long is the apprenticeship?
 [asked by @ZooeyMiller in #2](https://github.com/dwyl/apprenticeship/issues/2)
 
-Initially for 4 weeks.
+Initially for **8 weeks**. <br />
+Progress should be reviewed _weekly_. (_we need a system for this ..._) <br />
+<!--
+If both **you** and **we** (_the core @dwyl team member mentoring you_)
+feel you are making good progress,
+the apprenticeship can be extended.
+-->
+More details in the Apprenticeship Agreement.
+
+
+### Q: What is a "Previous Contribution" to a @dwyl Project?
+
+Contribution to a dwyl project can range from:
++ **Asking** a question (_seeking clarification_) in a GitHub Issue.
```

Comment:
And preferably opening a PR to ensure it is clear for others once you have an answer? Just asking questions is easy. Understanding the answer and doing something to help others is what shows commitment.

Reply:
@iteles agreed. please feel free to add this improvement. 👍